go-chaos/worker/chaos_worker.go

```diff
@@ -0,0 +1,76 @@
+// Copyright 2022 Camunda Services GmbH
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package worker
+
+import (
+	"context"
+	"time"
+
+	"github.com/camunda/zeebe/clients/go/v8/pkg/entities"
+	"github.com/camunda/zeebe/clients/go/v8/pkg/worker"
+)
+
+var CommandRunner func([]string, context.Context) error
```

Comment:
:question: This is a globally shared, mutable variable, right? I'm not sure if it's idiomatic in go but I'd have expected a `ChaosWorker` struct that encapsulates the handler function. Just to get rid of the global var.

Reply:
Yeah I was not 100% sure where to put it then. But I guess I need a wrapper around the handler to insert more inputs.

Reply:
Good point @oleschoenburg I fixed it :)